pimpmycause/profiles/migrations/0002_auto_20160730_2059.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.8 on 2016-07-30 20:59
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('profiles', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='pimpuser',
+            name='linkedin',
+            field=models.CharField(default='http://twitter.com', max_length=30),
```

Comment:
oh wow, thanks for your time explaining all this. I appreciate :)

Give me a bit more time to review please.